_posts/2019-07-21-introducing-rico.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  'Introducing Rico'
+author: markus
+featuredImage: 
+excerpt: 'Introduction to Rico, the application framework.'
+permalink: '/rico/2019/07/16/introducing-rico.html'
+categories: [Java, Jakarta EE, Java EE, Spring, Rico, Remoting, Monitoring, Security, Angular, WebComponents, JavaFX, Projector]
+header:
+  text: Introducing <span class="my-karakun">Rico<span>
+  image: post
+---
+
+[Rico](/rico) is an application framework fostered by <span class="my-karakun">Karakun</span> to help building better solutions in less time.
+
+Since [Hendrik blogged recently](/java/rico/2019/01/15/rico-server-timing.html) about a in-depth topic on [Rico](/rico), I thought we should give it an introduction post since it has not surfaced much so far....
+
+## History
+
+[Rico](/rico) originates from the _Canoo Dolphin Platform_. 
```

Comment:
the thing is known as Dolphin Platform not as Canoo DP

Reply:
wording changed